sycl/plugins/unified_runtime/ur/ur.hpp

```diff
@@ -356,13 +355,13 @@ class UrReturnHelper {
 
 // Global variables for ZER_EXT_RESULT_ADAPTER_SPECIFIC_ERROR
 constexpr size_t MaxMessageSize = 256;
-extern thread_local zer_result_t ErrorMessageCode;
+extern thread_local ur_result_t ErrorMessageCode;
 extern thread_local char ErrorMessage[MaxMessageSize];
 
 // Utility function for setting a message and warning
 [[maybe_unused]] void setErrorMessage(const char *message,
-                                      zer_result_t error_code);
+                                      ur_result_t error_code);
 
 // Returns plugin specific error and warning messages
 // TODO: promote to Unified Runtime API
-zer_result_t zerPluginGetLastError(char **message);
+ur_result_t zerPluginGetLastError(char **message);

```

Comment:
I think you don't need this function anymore. You can just implement `urGetLastResult`.
Feel free to ignore this comment if this is something you plan to do in next PRs.

Reply:
Thanks @igchor . good point. will remove->implement in follow up patch.